src/EnergyPlus/DaylightingDevices.cc

```diff
@@ -1391,26 +1389,26 @@ namespace DaylightingDevices {
 
         // Get the transmittance of each component and of total TDD
         switch (RadiationType) {
-        case DataDaylightingDevices::RadType::VisibleBeam: {
+        case RadType::VisibleBeam: {
             transDome = POLYF(COSI, state.dataConstruction->Construct(constDome).TransVisBeamCoef);
             transPipe = InterpolatePipeTransBeam(state, COSI, state.dataDaylightingDevicesData->TDDPipe(PipeNum).PipeTransVisBeam);
             transDiff = state.dataConstruction->Construct(constDiff).TransDiffVis; // May want to change to POLYF also!
 
             TransTDD = transDome * transPipe * transDiff;
 
         } break;
-        case DataDaylightingDevices::RadType::SolarBeam: {
```

Comment:
No need for namespace scoping here since `DataDaylightingDevices` and `DaylightingDevices` are now a single namespace.